AsterNET.ARI/ARIClient.cs

```diff
@@ -76,13 +78,24 @@ public ConnectionState ConnectionState
         /// <param name="endPoint"></param>
         /// <param name="application"></param>
         /// <param name="subscribeAllEvents">Subscribe to all Asterisk events. If provided, the applications listed will be subscribed to all events, effectively disabling the application specific subscriptions.</param>
```

Comment:
Seem to be missing descriptions for new params

Reply:
will do